app/controllers/idv/hybrid_mobile/capture_complete_controller.rb

```diff
@@ -0,0 +1,49 @@
+module Idv
+  module HybridMobile
+    class CaptureCompleteController < ApplicationController
+      include IdvSession
+      include IdvStepConcern
+      include StepIndicatorConcern
+      include StepUtilitiesConcern
+
+      before_action :render_404_if_hybrid_mobile_controllers_disabled
```

Comment:
I had the thought that we should add a before action to make sure document capture is complete, but then realized that is getting ahead of ourselves since that step is not implemented yet.